azkaban-web-server/src/web/js/azkaban/view/job-list.js

```diff
@@ -240,6 +247,15 @@ azkaban.JobListView = Backbone.View.extend({
     evt.stopImmediatePropagation();
   },
 
+  handleToggleMenuExpandAll: function (evt) {
```

Comment:
Is it possible to implement the opposite: Collapse All?

Reply:
I was thinking about that and, yes it would be possible but I don't think it's desirable.

1. You can always collapse a menu by clicking the single arrow. If you do re-open that menu all menus will still be expanded though.

2. A menu may be in a partially expanded state and you may still want to expand the remainder of the unexpanded children, so changing expand all to collapse all upon opening would remove that ability.

Reply:
Talked offline, we'll do only "expand all" for now. If it's confusing for users we'll revisit this.